hydra-module.nix

```diff
@@ -169,6 +169,12 @@ in
         example = [ "/etc/nix/machines" "/var/lib/hydra/provisioner/machines" ];
         description = "List of files containing build machines.";
       };
+
+      useSubstitutes = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to use binary caches for downloading store paths.";
```

Comment:
Could you add implications from https://github.com/NixOS/hydra/issues/297#issuecomment-211918715 ?